mitmproxy/tools/main.py

```diff
@@ -2,11 +2,11 @@
 
 import sys
 
-if sys.version_info < (3, 5):
+if sys.version_info < (3, 6):
     # This must be before any mitmproxy imports, as they already break!
     # Keep all other imports below with the 'noqa' magic comment.
     print("#" * 49, file=sys.stderr)
-    print("# mitmproxy only supports Python 3.5 and above! #", file=sys.stderr)
+    print("# mitmproxy only supports Python 3.6! #", file=sys.stderr)
```

Comment:
why skip the "and above"?

Reply:
Unless we start testing for 3.7 I think we should not claim that it works for "and above".
Felt kinda wrong, semver and everything... But I can revert it if you have strong feeling about it.

Reply:
Yes - once 3.7 is supposed to be backward compatible, we have no particular tie-ins to 3.6.